mono/metadata/domain.c

```diff
@@ -526,7 +526,6 @@ mono_init_internal (const char *filename, const char *exe_filename, const char *
 #endif
 
 #ifndef HOST_WIN32
-	mono_w32handle_init ();
 	wapi_init ();
 #endif
```

Comment:
Why is this needed ? Traditionally, we tried to move away from lazy initialization since it complicates things.